runtime/catalog-karapace/src/main/java/io/aklivity/zilla/runtime/catalog/karapace/internal/KarapaceCatalogHandler.java

```diff
@@ -271,15 +334,10 @@ private String sendHttpRequest(
             HttpResponse<String> httpResponse = client.send(httpRequest, HttpResponse.BodyHandlers.ofString());
             boolean success = httpResponse.statusCode() == 200;
             String responseBody = success ? httpResponse.body() : null;
-            if (!success)
-            {
-                event.remoteAccessRejected(catalogId, httpRequest, httpResponse.statusCode());
-            }
             return responseBody;
         }
         catch (Exception ex)
         {
-            event.remoteAccessRejected(catalogId, httpRequest, 0);
         }
```

Comment:
Rather than having an empty catch block, I think it would better document the intent if `responseBody` is hoisted outside the `try/catch` and explicitly set to `null` in the `catch` before `return responseBody;` at the end of the method.